boards/arm/nucleo_wl55jc/nucleo_wl55jc_defconfig

```diff
@@ -0,0 +1,43 @@
+CONFIG_SOC_SERIES_STM32WLX=y
+CONFIG_SOC_STM32WL55XX=y
+# 48MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=48000000
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# enable pinmux
+CONFIG_PINMUX=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# clock configuration
+CONFIG_CLOCK_CONTROL=y
+# SYSCLK selection
+CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
+# use HSE as PLL input
+CONFIG_CLOCK_STM32_PLL_SRC_HSI=y
+
+# HSE is prersent on nucleo board (32MHz), but reserved for further LoRa use
```

Comment:
`# HSE is present`   --> # HSE is present

Reply:
Fixed